src/Styles.jsx

```diff
@@ -1,10 +1,16 @@
 export const ModalStyle = {
   overlay: {
-    position: 'fixed',
-    top: '15%',
-    left: '25%',
-    right: '25%',
-    bottom: '25%',
+    position: 'absolute',
+    top: '0%',
+    left: '0%',
+    right: '0%',
+    bottom: '0%',
   },
-  content: {}
+  content: {
```

Comment:
I made the content within the modal fixed, otherwise you can't resize the overlay without repositioning the modal itself.